app/views/groups/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render @groups %>
```

Comment:
I think you need to wrap this with a `<table>...</table>` tabs as a minimum.
Would be also good to include a table header row

Reply:
I was going to do everything in the next PR for the group#index. Here I have just the minimum to test my group#create. Do you think it is ok or it's better to do here? Thanks

Reply:
It is not correct markup to use `tr` and `td` without `table` tags

Reply:
Fair enough. Thanks!
Done with fixup on fe52ac6